GameMetadata.Net/MetadataClient.cs

```diff
@@ -6,26 +6,28 @@
 using System.Text;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 using Newtonsoft.Json.Serialization;
 
 namespace GameMetadata.Net
 {
     public class MetadataClient
     {
-        private static readonly JsonSerializerSettings DefaultSettings = new JsonSerializerSettings
+        private const string ApiEndpoint = "http://www.steamgriddb.com/search.php";
+        
+        private readonly static JsonSerializerSettings DefaultSettings = new JsonSerializerSettings
         {
             ContractResolver = new DefaultContractResolver
             {
                 NamingStrategy = new SnakeCaseNamingStrategy()
             }
         };
 
-        protected static T DeserializeObject<T>(string json) =>
-            JsonConvert.DeserializeObject<T>(json, DefaultSettings);
+        private static T DeserializeObject<T>(string json) 
+            => JsonConvert.DeserializeObject<T>(json, DefaultSettings);
+
         public async Task<SearchedGame> GetGameMetadataAsync(string query)
         {
-            var images = await GetAllGameImagesAsync(query);
+            IEnumerable<GameImage> images = await GetAllGameImagesAsync(query);
```

Comment:
lmao stop ruining my `var`s XD

Reply:
ahahah lol

Reply:
a. it's just that.
b. why not? - it's not that evident that's it returns an ienumerable

Reply:
well `AllGamesImages` since it's plural you expect it to be enumerable lol